dataprofiler/profilers/profile_builder.py

```diff
@@ -602,37 +604,32 @@ def update_profile(self, data, sample_size=None, min_true_samples=None):
         :type min_true_samples
         :return: None
         """
-        if not sample_size:
-            sample_size = self._samples_per_update
```

Comment:
I think we should first check the data type, then get samples

Reply:
updated